app/views/contexts/list_context.py

```diff
@@ -44,6 +48,7 @@ def __call__(
     def _build_list_items_context(
```

Comment:
enforcing keyword args for this might be useful

Reply:
Enforced 👍